scripts/migrate-to-pnpm-10.js

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env node
+
+const { execSync } = require('child_process');
```

Comment:
Read the target version from `package.json` so that this can take it and do the rest.